hector_geotiff/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.0.2)
 project(hector_geotiff)
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Ideally, the enforcement of the c++ standard is removed completely:
```suggestion
```
The default of current compilers (g++ and clang++) is C++17 anyway (Ubuntu >= 22.04).